openlcs/libs/kojiconnector.py

```diff
@@ -483,3 +489,64 @@ def download_source(self, build):
             err_msg = f'Failed to download source. Reason: {err}'
             raise RuntimeError(err_msg) from None
         return temp_dir
+
+    def get_module_mapping_SRPMs(self, module_nvr):
+        """
+        Return module SRPM NVRs.
+        """
+        build = self.get_build(module_nvr)
+        tag = build.get('extra').get('typeinfo').get('module').get(
+                'content_koji_tag')
+        try:
+            builds = self.list_tagged(tag)
+        except RuntimeError as err_msg:
+            err_msg = f'Failed to get the module {module_nvr} SRPM nvrs'
+            raise RuntimeError(err_msg) from None
+        package_nvrs = []
+        for build in builds:
+            package_nvrs.append(build.get('nvr'))
+        return package_nvrs
+
+    @staticmethod
+    def get_component_flat(data, comp_type):
+        return {
+            'uuid': str(uuid.uuid4()),
+            'type': comp_type,
+            'name': data.get('name'),
+            'version': data.get('version'),
+            'release': data.get('release'),
+            'summary_license': '',
+            "arch": data.get("arch", ""),
+            'synced': False
+        }
+
+    def get_module_component(self, module_nvr):
+        """
+        Return module component itself.
+        """
+        nvr = koji.parse_NVR(module_nvr)
+        module_component = [self.get_component_flat(nvr, 'RPMMOD')]
+        return module_component
+
+    def get_module_srpm_components(self, module_nvr):
+        """
+        Get module child srpm components.
+        """
+        srpm_nvrs = self.get_module_mapping_SRPMs(module_nvr)
```

Comment:
case sensitive of function name mismatch with others

Reply:
Updated.